bookstore/handlers.py

```diff
@@ -85,11 +86,17 @@ def _publish(self, model, path):
 
         # Return 501 - Not Implemented
         # Until we're ready
-        self.set_status(501)
+        # self.set_status(501)
+        self.set_status(201)
+        resp_content = {
+            "s3path": full_s3_path,
+            # "versionID": obj['VersionId'],
```

Comment:
I think it's ok if we only return the `versionId` if available, at least to make things work well with minio.

Reply:
Sounds reasonable! We could also write the files with versionId as a query param and I think it might “just work” but that’s so different from actually versioning with the Bucket that I wasn’t sure if that made sense (since we won’t do that client side in prod).

Reply:
The version Id is created by Amazon, this is only for response when we publish to the same spot.